lib/commands/addJob-6.lua

```diff
@@ -47,7 +48,15 @@ else
   jobId = ARGV[2]
   jobIdKey = ARGV[1] .. jobId
   if rcall("EXISTS", jobIdKey) == 1 then
-    return jobId .. "" -- convert to string
+    if ARGV[12] == '1' then
```

Comment:
So basically what this is doing is creating a new jobId (with postfix buffer) if the current jobId is being used. So in practice is like supporting 2 jobs with the same jobId instead of one... I am not sure I like this solution to be honest.